app/javascript/listings/listings.jsx

```diff
@@ -1,6 +1,11 @@
 import { h, Component } from 'preact';
+import debounce from 'lodash.debounce';
+import { fetchSearch } from '../src/utils/search';
 import SingleListing from './singleListing';
 
+/* How many listings to show per page */
```

Comment:
shouldn't we use jsdoc style comments here?

Reply:
Updated!